src/core/textures/TextureUvs.js

```diff
@@ -27,6 +27,7 @@ export default class TextureUvs
         this.y3 = 1;
 
         this.uvsUint32 = new Uint32Array(4);
+        this.uvsMatrix = null;
```

Comment:
Is this required? Seems like this would not be commonly used by the core of pixi. We should remove.